os/fs/procfs/fs_procfs.c

```diff
@@ -166,6 +167,10 @@ static const struct procfs_entry_s g_procfsentries[] = {
 	{"ereport/*", &ereport_operations},
 #endif
 
+#if defined(CONFIG_PM_METRICS) && !defined(CONFIG_FS_PROCFS_EXCLUDE_PMINFO)
```

Comment:
CONFIG_FS_PROCFS_EXCLUDE_PMINFO is enough. Because FS_PROCFS_EXCLUDE_PMINFO checks PM_METRICS.

Reply:
Even if CONFIG_PM_METRICS is disabled, the default value of CONFIG_FS_PROCFS_EXCLUDE_PMINFO is "n".
So we should check CONFIG_PM_METRICS, otherwise "pminfo_operations" is not defined.
"pminfo_operations" in "fs_procfspminfo.c" can be defined when CONFIG_PM_METRICS is enabled.